library/src/main/assets/migrations/1_SETUP.SQL

```diff
@@ -1,15 +1,15 @@
 CREATE TABLE IF NOT EXISTS 'request' (
                                 _id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
                                 request_timestamp TEXT NOT NULL,
-                                UNIQUE (request_timestamp) ON CONFLICT REPLACE
+                                UNIQUE (request_timestamp) ON CONFLICT REPLACE --TODO: should abort this - else risk two downloads with same ID in DatabaseInteractions
```

Comment:
👍 would be good to move all of them to issues and perhaps drop all the v1 issues as well